build/meson/tests/meson.build

```diff
@@ -112,13 +112,15 @@ invalidDictionaries = executable('invalidDictionaries',
   dependencies: [ libzstd_dep ],
   install: false)
 
-legacy_sources = [join_paths(zstd_rootdir, 'tests/legacy.c')]
-legacy = executable('legacy',
-  legacy_sources,
-  # Use -Dlegacy_level build option to control it
-  #c_args: '-DZSTD_LEGACY_SUPPORT=4',
-  dependencies: [ libzstd_dep ],
-  install: false)
+if legacy_level <= 4
```

Comment:
In zstd legacy 0 means disabled. If it is the same in meson then we need `if legacy_level > 0 && legacy_level <= 4`.

Reply:
Done.